### DISCUSSION TITLE
Regression producing PDF from TIFF with Group4 compression regarding colorspace?

### DISCUSSION THREAD

**Discussion by meinhardh** [ImageMagick/ImageMagick#4089] [Help] [answered]
I am using ImageMagick on a Ubuntu 20.04 system with the standard configuration. The ImageMagick version is 6.9.10.23+dfsg-2.1ubuntu11.4.

I have produced a one-page PDF document from a TIFF file with the following command line:
`$ convert -density 300 p-000.tif -type bilevel -compress Group4 -colorspace Gray p-000.pdf`

The resulting file shows a colorspace of icc:

`$ pdfimages -list p-000.pdf`
`page   num  type   width height color comp bpc  enc interp  object ID x-ppi y-ppi size ratio`
`  1     0 image    2480  3507  icc     1   1  ccitt  no         8  0   300   300  110K  10%`

This colorspace is a problem for some PDF readers - not finding a color index they show everything in the same color.

In July 2020 I had the same problem with the above command line without the "-colorspace Gray" option. Adding the option fixed the problem back then, it made the colorspace move from icc to gray, and all readers were happy. Now adding the option or not does not seem to make any difference, the colorspace of the resulting PDF is always icc.

Is this a regression/bug? How can I get a PDF with the intended colorspace?

**Comment by fmw42** [comment]
try adding -type grayscale.

**Comment by meinhardh** [comment]
Thanks, but nope - same result, colorspace is icc.

**Comment by fmw42** [comment]
You could try adding a gray profile.

**Comment by fmw42** [comment]
What does exiftool say about your colorspace?  Can you post your input and output images so we can examine and test ourselves.  What software are you trying to pass the output to that objects?

**Comment by snibgo** [comment, marked as answer]
Does p-000.tif have an embedded ICC profile? If it does, then "-strip" it.

**Reply by meinhardh** [reply]
And as @fmw42 pointed out, exiftool is the tool of choice to tell whether or not there is an embedded profile.

**Comment by meinhardh** [comment]
Thanks - will provide the files tonight when I have access to the machine again.

**Comment by meinhardh** [comment]
@snibgo was right - indeed the TIFF files had embedded profiles. Adding '-strip' fixed the problem. Thanks all for your help - issue solved.
